horton/scripts/wpart.py

```diff
@@ -77,25 +76,26 @@ def wpart_slow_analysis(wpart, mol):
         # Prepare solid harmonics on grids.
         grid = wpart.get_grid(index)
         if wpart.lmax > 0:
-            work = np.zeros((grid.size, npure-1), float)
-            work[:,0] = grid.points[:,2] - wpart.coordinates[index,2]
-            work[:,1] = grid.points[:,0] - wpart.coordinates[index,0]
-            work[:,2] = grid.points[:,1] - wpart.coordinates[index,1]
+            work = np.zeros((grid.size, npure - 1), float)
```

Comment:
No problem. I'm fine either way. I was only worried that it would take extra efforts to add these spaces, but that is clearly not a concern. (It is rather the opposite.)